careers/__init__.py

```diff
@@ -5,6 +5,13 @@
 minPythonVersion = (3, 9)
 minPythonVersionStr = '.'.join([str(x) for x in minPythonVersion])
 
+import sys  
```

Comment:
I have no idea how to do this "correctly". The only way I could get it to run in flask is to import the packages (as directories). I found this on the web so take it with a grain of salt. If you know a better way, let me know.

Reply:
Although with this I can debug it as a flask app in visual studio code.